### ISSUE TITLE
Use AppIcon from Assets generated by go-bindata

### ISSUE THREAD

**Issue by aashayshah** [asticode/go-astilectron#27]
Is it possible to do use assets linked with the binary assets generated by go-bindata for app icon logo?
Currently I have to give a file path but I would like it to use the asset file generated by go-bindata.

So, for example:
```
var a *astilectron.Astilectron
	if a, err = astilectron.New(astilectron.Options{
		AppName:            "App v0.1",
		AppIconDefaultPath: "vendor/logo.png",
		AppIconDarwinPath:  "vendor/logo.icns",
		BaseDirectoryPath:  "./",
	}); err != nil {
		astilog.Fatal(errors.Wrap(err, "creating new astilectron failed"))
	}
	a.SetProvisioner(astilectron.NewDisembedderProvisioner(Asset, "vendor/astilectron-v"+astilectron.VersionAstilectron+".zip", "vendor/electron-v"+astilectron.VersionElectron+".zip"))
```
But here vendor is a directory, but if we can pass the Asset Directly instead of path, or any other way then it would be great to have app icon embedded in a single file especially when you are building a single file distribution application.

**Comment by veecue** [comment]
Since the path is passed directly to electron, I don't think, that that is possible. My current solution is to write the AppIcons to os.TempDir()

**Comment by asticode** [comment]
@aashayshah I don't think this is a good idea to put your custom resources in the `vendor` folder. It may look simpler because you're actually taking advantage of `astilectron`'s embedder feature but you'll get stuck pretty quickly as you are now.

If you take a look at the bootstrap, you'll see that in order to to embed your custom assets, you'll have to add your custom provision logic on top of `astilectron`'s Provisioner : assets: https://github.com/asticode/go-astilectron/blob/master/bootstrap/run.go#L35.

Basically `astilectron`'s Provider embeds a `vendor` directory that contains basic dependencies, whereas your provisioner should embed a `resources` directory that contains your custom resources.

Once again, I'd suggest you take a look at the bootstrap that disembed custom resources and use them in the process.

Does that answer your question?

Cheers

**Comment by aashayshah** [comment]
@asticode Yes, you are right, so I am closing this issue.

Thanks.
